examples/confidential-block-building/README.md

```diff
@@ -0,0 +1,98 @@
+### Build 
```

Comment:
Maybe add an intro paragraph outlining what this example does?

Reply:
@metachris please review